actions/kubectl-actions.py

```diff
@@ -24,7 +24,7 @@ def _kubectl(args):
 
 def get_kubeconfig():
     """
-    Read the kubeconfig on the master and return it as JSON
+    Read the kubeconfig on the control-plane leader and return it as JSON
```

Comment:
"Leader" could be misleading within the context of Juju charm development. This isn't getting config from the leader, it's getting config from the local unit.